### ISSUE TITLE
node-uuid deprecation warning

### ISSUE THREAD

**Issue by jindrichmynarz** [linkedpipes/etl#500]
Running front-end produces a deprecation warning:

```
npm WARN deprecated node-uuid@1.4.0: Use uuid module instead
```

Tested using the develop branch 654d3d6e04dcba5c86afae69e5d42842c02729c9 (npm v5.4.2, node v8.7.0).

**Comment by jakubklimek** [comment]
related to https://github.com/linkedpipes/etl/issues/496

**Comment by skodapetr** [comment]
Fixed in #504
